cli/tests/integration/npm_tests.rs

```diff
@@ -449,6 +449,16 @@ itest!(permissions_outside_package {
   http_server: true,
 });
 
+itest!(run_existing_npm_package {
+  args: "run --allow-read --node-modules-dir npm:@denotest/bin",
+  output: "npm/run_existing_npm_package/main.out",
+  envs: env_vars_for_npm_tests(),
+  http_server: true,
+  temp_cwd: true,
+  cwd: Some("npm/run_existing_npm_package/"),
+  copy_temp_dir: Some("npm/run_existing_npm_package/"),
+});
```

Comment:
It would be good to add a test for doing something like `npm:@denotest/bin/cli-esm` as well (given that this bug exists in the current implemention)

Reply:
added a test case with subpath now, thanks ~